Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -321,16 +252,140 @@ - (void)_stealFirstResponder;
 
 - (void)_showBugTitleCaptureAlert;
 {
-    UIAlertView *bugTitleCaptureAlert = [[UIAlertView alloc] initWithTitle:@"What Went Wrong?" message:@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later." delegate:self cancelButtonTitle:@"Cancel" otherButtonTitles:@"Compose Report", nil];
-    bugTitleCaptureAlert.alertViewStyle = UIAlertViewStylePlainTextInput;
+    /*
+     iOS 8 often fails to transfer the keyboard from a focused text field to a UIAlertView's text field.
+     Transfer first responder to an invisble view when a debug screenshot is captured to make bug filing itself bug-free.
+     */
+    [self _stealFirstResponder];
     
-    UITextField *bugTitleTextField = [bugTitleCaptureAlert textFieldAtIndex:0];
-    bugTitleTextField.autocapitalizationType = UITextAutocapitalizationTypeSentences;
-    bugTitleTextField.autocorrectionType = UITextAutocorrectionTypeYes;
-    bugTitleTextField.spellCheckingType = UITextSpellCheckingTypeYes;
-    bugTitleTextField.returnKeyType = UIReturnKeyDone;
+    NSString * const title = NSLocalizedString(@"What Went Wrong?", nil);
+    NSString * const message = NSLocalizedString(@"Please briefly summarize the issue you just encountered. You’ll be asked for more details later.", nil);
+    NSString * const composeReportButtonTitle = NSLocalizedString(@"Compose Report", nil);
+    NSString * const cancelButtonTitle = NSLocalizedString(@"Cancel", nil);
```

Comment:
One last nit. Can you add quick description strings for the `NSLocalizedString`s above? Say:
"Title text for alert asking user to describe a bug they just encountered"
"Subtitle text for alert asking user to describe a bug they just encountered"
"Button title to compose bug report"
"Button title to not compose a bug report"

Feel free to modify. The above are just suggestions. Thanks for being so responsive!